### PR TITLE
Added vehicle width and height as an option for auto (and derived: taxi, bus, hov) profile

### PR DESCRIPTION
**Pull request by maksimandrianov**
# Issue

What issue is this PR targeting? If there is no issue that addresses the problem, please open a corresponding issue and link it here.

## Tasklist

 - [ ] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Update the docs with any new request parameters or changes to behavior described
 - [ ] Update the [changelog](CHANGELOG.md)
 - [ ] If you made changes to the lua files, update the [taginfo](taginfo.json) too.

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: src/sif/autocost.cc [hunk 1]
```diff
@@ -65,11 +65,17 @@ constexpr float kLeftSideTurnCosts[] = {kTCStraight,         kTCSlight,  kTCUnfa
 constexpr float kMinFactor = 0.1f;
 constexpr float kMaxFactor = 100000.0f;
 
+// Default auto attributes
+constexpr float kDefaultAutoHeight = 1.6f; // Meters (62.9921 inches)
+constexpr float kDefaultAutoWidth = 1.4f;  // Meters (55.1181 inches)
```
Comment (kevinkreiser) [inline src/sif/autocost.cc:70]: this should be the original value of 1.9. 1.4 is incredibly narrow!

**Comment by kevinkreiser** [comment]
it looks like you reverted submodule changes

File: src/sif/autocost.cc (hunk 1 above)
Comment (maksimandrianov) [reply src/sif/autocost.cc]: Okey, fixed

**Comment by maksimandrianov** [comment]
> it looks like you reverted submodule changes

it is a mistake, thanks!

File: test/gurka/test_permissible_dimensions.cc [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  for (const auto& costing : kSupportedCostingTypes) {
+    const auto result = gurka::do_action(valhalla::Options::route, map, {"A", "I"}, costing,
+                                         {{"/costing_options/" + costing + "/width", "2.0"}});
+    ASSERT_EQ(result.trip().routes(0).legs_size(), 1);
+    gurka::assert::raw::expect_path(result, {"AB", "BC", "CD", "DE", "EF", "FG", "GH", "HI"});
+  }
+
+  for (const auto& costing : kSupportedCostingTypes) {
+    const auto result = gurka::do_action(valhalla::Options::route, map, {"A", "I"}, costing,
+                                         {{"/costing_options/" + costing + "/width", "1.8"}});
```
Comment (gknisely) [inline test/gurka/test_permissible_dimensions.cc:45]: @maksimandrianov can we get a test with 1.9.

Comment (maksimandrianov) [reply test/gurka/test_permissible_dimensions.cc:45]: Yes, I have added

**Review by gknisely — APPROVED** [review]

**Pull request merged**